Yosemite/Yosemite/Model/Payments/CardPresentPaymentsConfiguration.swift

```diff
@@ -62,6 +62,22 @@ public struct CardPresentPaymentsConfiguration: Equatable {
                 contactlessLimitAmount: nil,
                 minimumOperatingSystemVersionForTapToPay: .init(majorVersion: 16, minorVersion: 0, patchVersion: 0)
             )
+        case .PR:
+            self.init(
+                countryCode: country,
+                paymentMethods: [.cardPresent],
+                currencies: [.USD],
+                paymentGateways: [WCPayAccount.gatewayID, StripeAccount.gatewayID],
+                supportedReaders: [.chipper, .stripeM2],
+                supportedPluginVersions: [
+                    .init(plugin: .wcPay, minimumVersion: "9.0.0"),
+                    .init(plugin: .stripe, minimumVersion: "6.2.0")
```

Comment:
Do we really support Stripe or should we apply the same fix as we did with WooPayments?

```suggestion
```

On your test store, I added a Stripe extension but the card reader connection fails.

Reply:
> Do we really support Stripe or should we apply the same fix as we did with WooPayments?

That's a good question, I've only scoped these changes to WooCommerce Payments. I'm unsure who's currently tackling changes in the Stripe plugin and if we should handle this as well, I've asked here: p1741748917720889?thread_ts=1739205177.793219&cid=CGGCLBN58-CGGCLBN58

Reply:
After several attempts this last week with wcpay folks ( p1739205177793219-ref-CGGCLBN58 ) it seems that the Stripe plugin will need dedicated time to make this work, I've [opened an issue on the plugin repo](https://href.li/?https://github.com/woocommerce/woocommerce-gateway-stripe/issues/4061) and removed support for the Stripe plugin in this PR: https://github.com/woocommerce/woocommerce-ios/pull/14972/commits/b457b4f71b592cfe7f1f70770486d378d7a67564